lib/api/request/requestResponse.ts

```diff
@@ -154,7 +159,11 @@ export class RequestResponse {
 
   constructor (request) {
     this.raw = false;
-    this[_request] = request;
+
+    Reflect.defineProperty(this, _request, {
```

Comment:
Here, `_request` is still `request\u200b`, maybe you wanted to do 
```suggestion
    Reflect.defineProperty(this, 'request', {
```